azavea/raster-foundry
# Feature/lol/new status format

Connects #259 

This is a fairly major overhaul of the statuses and errors in layer and layerimage models.
Statuses are now split into two fields a start field and an end field. The UI does not/will not care about these as they are returned as JSON as simple Booleans. Each status also has an associated error message field.

### To test
 * Run migrations and restart the worker.
 * Kick off a normal job flow by uploading a file.
 * In the admin UI refresh the page as the job move through the pipeline.
 * Note that timestamps are added at the start and end of each part of the job.
